docs/operations/hotplug_interfaces.md

```diff
@@ -0,0 +1,102 @@
+will be attached to the restarted VM. If these interfaces are persisted, they
+become part of the VM spec - i.e. will not be considered hotplugged.
+
+## Enabling network interface hotplug support
+Network interface hotplug support must be enabled via a
+[feature gate](https://kubevirt.io/user-guide/operations/activating_feature_gates/#how-to-activate-a-feature-gate).
+The feature gates array in the KubeVirt CR must feature `HotplugNICs`.
+
+## Requirements
+Adding an interface to a KubeVirt Virtual Machine requires first an interface
+to be added to a running pod. This is not trivial, and has some requirements:
+
+- [multus dynamic networks controller](https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller):
+  this daemon will listen to annotation changes, and trigger multus to configure
+  a new attachment for this pod.
+- multus running as a [thick plugin](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/thick-plugin.md):
+  this multus version exposes an endpoint to create attachments for a given pod
+  on demand.
+
+## Adding an interface to a running VMI
+First start a VMI - e.g.
+[examples Fedora VMI](https://github.com/kubevirt/kubevirt/blob/main/examples/vmi-fedora.yaml).
+
+You should configure a network attachment definition - where the pod interface
+configuration is held. The snippet below shows an example of a very simple one:
+```yaml
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: new-fancy-net
+spec:
+    config: '{
+      "cniVersion": "0.3.1",
+      "type": "bridge",
+      "mtu": 1300,
+      "name":"new-fancy-net"
+    }'
+```
+
+Please refer to the
+[multus documentation](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/how-to-use.md#create-network-attachment-definition)
+for more info.
+
+Once the virtual machine is running, and the attachment configuration
+provisioned, the user can request the interface hotplug operation. Please refer
+to the following snippet:
+```bash
+virtctl addinterface vmi-fedora --network-name new-fancy-net --iface-name dyniface1
+```
+
+**NOTE**: You can use the `--help` parameter for more information on each
+parameter.
+
+You can now check the VMI status for the presence of this new interface:
+```bash
+kubectl get vmi vmi-fedora -ojsonpath="{ @.status.interfaces }"
+```
+
+### Virtio Limitations
+The hotplugged interfaces have `model: virtio`. This imposes several
+limitations: each interface will consume a PCI slot in the VM, and there are a
+total maximum of 32. Furthermore, other devices will also use these PCI slots
+(e.g. disks, guest-agent, etc).
+
+For the `q35` machine type, the issue is even worse: the users will be able to
+hotplug a **single** network interface into the guest. You can find more
+information in the
+[libvirt documentation](https://libvirt.org/pci-hotplug.html#q35-machine-type).
+
+Ways to improve this are being thought of, enabling users to hotplug more than
```

Comment:
User-guide is a poor to-do list. I would not keep this note in the user-guide unless you have a link to a known limitation (GitHub Issue) with a backlink